docs/map-making/upgrading-1-8-maps.md

```diff
@@ -13,48 +13,38 @@ Note: Some items are not validated; thus, if your game plays on 1.9.0.0.3266, th
 ### Not already deprecated xml changes:
 _(you normally need to change these only)_
 
-`attatchment`:`attachment`
-`Attatchment`:`Attachment`
-`attatchTo=`:`attachTo=`
-`="isImpassible"`:`="isImpassable"`
-`="conditionType" value="XOR"`:`="conditionType" value="1"`
-`="turns":="rounds"`
-`="Battleships repair at end of round"`:`="Units Repair Hits End Turn"`
-`="Battleships repair at beginning of round"`:`="Units Repair Hits Start Turn"`
-`="Naval Bombard Casualties Return Fire Restricted"`:`="Naval Bombard Casualties Return Fire"`
+- `attatchment`:`attachment`
+- `Attatchment`:`Attachment`
+- `attatchTo=`:`attachTo=`
+- `="isImpassible"`:`="isImpassable"`
+- `="conditionType" value="XOR"`:`="conditionType" value="1"`
+- `="turns":="rounds"`
+- `="Battleships repair at end of round"`:`="Units Repair Hits End Turn"`
+- `="Battleships repair at beginning of round"`:`="Units Repair Hits Start Turn"`
+- `="Naval Bombard Casualties Return Fire Restricted"`:`="Naval Bombard Casualties Return Fire"`
 
 ### Special code xml changes for the above changes:
 _(you probably don't have these)_
 
-`value="conditionType" count="XOR"`:`value="conditionType" count="1"`
-`value="conditionType" count="-reset-XOR"`:`value="conditionType" count="-reset-1"`
+- `value="conditionType" count="XOR"`:`value="conditionType" count="1"`
```

Comment:
@Cernelius , as far as I can tell this was never done and "1" is not an equivalent value. Do you have background for why this line item was added? Do you know of any maps where this update was actually made? It looks like the code still only handles 'xor'.

Reply:
I remember you told me that the "XOR" is not anymore supported. If the "XOR" is still supported and it is not deprecated, then all the items about converting "XOR" to "1" should be removed from the document.

The reason why I suggested to remove the "XOR" is that "XOR" means "1", as long as the conditions are 2 or more (I don't know what's going to happen if you have "XOR" and 1 or 0 conditions (logically, you should never have "XOR" with less than 2 conditions), but I assume it is still the same as having "1"). Therefore, the "XOR" value is completely useless, as you can use "1" instead of it. I also suppose not everyone knows what "XOR" means, especially since in English "or" is normally supposed to be exclusive ("You can have this or that for 1 pound" doesn't mean you can have both).

Reply:
Thanks for the response here. Yeah, I think removing the XOR mention for now is best until we can clarify it. Agree that it's a classic problem that the commonly used 'or' is not a logical or, and is actually a logical 'xor'.

I was trying to track down the '1' value that was mentioned here and noticed that we still had code to handle xor.

Reply:
So, can you confirm that the "XOR" has never been removed, thus the information about the fact that the "XOR" needed to be changed to "1" has always been wrong.

If so, can we at least deprecate the "XOR" in favour of "1"?

Reply:
XOR Is still there.

What would "1" mean as a condition type?

Reply:
A numeral condition type means that exactly that number of conditions must be true (no more, no less). That is why having "1" is the same as having "XOR".

Reply:
Thanks for the clarification @Cernelius 

For the purpose of this conversation, it seems like the XOR mention should be removed from this doc since as far as I can tell it has not been updated.

If we were to update it, it's probably a conversation to move to forums. IMO we could probably do even better than changing the value from condition type to condition count.